tests/unit_tests/c_wrappers/res/job_queue/test_ext_job.py

```diff
@@ -13,11 +14,15 @@ def test_load_forward_model_raises_on_missing():
     with pytest.raises(
         ConfigValidationError, match="Could not open job config file 'CONFIG_FILE'"
     ):
-        _ = ExtJob.from_config_file("CONFIG_FILE")
+        _ = ExtJob.from_config_file("CONFIG_FILE", use_new_parser=False)
 
+    with pytest.raises(IOError, match="file(.+?)not found"):
```

Comment:
We still want `from_config_file` to give `ConfigValidationError` as that means it is shown nicely in the GUI.